src/nginxconfig/i18n/ru/templates/domain_sections/presets.js

```diff
@@ -30,4 +30,5 @@ export default {
     frontend: 'Фронтэнд',
     nodeJs: 'Node.js',
     singlePageApplication: 'Одностраничное приложение',
+    rails: 'Рельсы',
```

Comment:
Is this the correct translation? Googling would suggest this is the translation for a train track rail, does that work for Rails as the software?